src/InitOMPasses.hpp

```diff
@@ -5,6 +5,10 @@
 #include "mlir/Pass/Pass.h"
 #include "src/Pass/Passes.hpp"
 
+#ifdef __NNPA__
```

Comment:
I don't think we should pollute the high level space with accelerator stuff. Each accelerator should take care of registering its own passes, down inside its own source tree.